main/libedit/template.py

```diff
@@ -17,6 +17,8 @@
 
 def post_install(self):
     self.install_license("COPYING")
+    # conflicts with readline-devel-man
+    self.uninstall("usr/share/man/man3/history.3")
```

Comment:
could otherwise rename it but idk, there are a ton of manpages and not renaming them also would be inconsistent in that case